inertia/pages/manager/authentication/validators/reset_password_validator.ts

```diff
@@ -0,0 +1,8 @@
+import { z } from 'zod'
+
+export const resetPasswordValidator = z.object({
+  password: z.string().min(2).max(50),
+  password_confirmation: z.string().min(2).max(50),
```

Comment:
Remove  `password_confirmation` and put `.confirmed()` method in `password` constraint declaration